x-pack/test/api_integration/apis/synthetics/sync_global_params.ts

```diff
@@ -287,8 +287,9 @@ export default function ({ getService }: FtrProviderContext) {
       const deleteResponse = await supertestAPI
         .delete(SYNTHETICS_API_URLS.PARAMS)
         .set('kbn-xsrf', 'true')
-        .send({ ids })
-        .expect(200);
+        .send({ ids });
+
+      expect(deleteResponse.status).eql(200, JSON.stringify(deleteResponse.body));
```

Comment:
I don't understand the change here. Seems like `.expect(200)` is more straightforward.